src/integrations/prefect-docker/pyproject.toml

```diff
@@ -71,7 +71,7 @@ fail_under = 80
 show_missing = true
 
 [tool.uv.sources]
-prefect = { path = "../../../" }
+prefect = { path = "../../../", editable = true }
```

Comment:
is this a change you intended to commit?

Reply:
Yeah, it makes making changes in both the core library and an integration library easier. This change only affects the development environment.